elf/module.go

```diff
@@ -221,6 +221,13 @@ func NewModule(fileName string) *Module {
 	return module
 }
 
+func NewModuleWithLogSize(fileName string, logSize int) *Module {
+	module := newModule()
+	module.fileName = fileName
+	module.log = make([]byte, logSize)
```

Comment:
Could you extract the default size as a constant, and pass the log size as an argument to `newModule`? That way we avoid doing an unnecessary allocation of `524288` bytes in the custom size case.

This also doesn't allow setting a custom size when using `NewModuleFromReader`, if that is a concern.

Reply:
Done ! 👍